src/olympia/lib/git.py

```diff
@@ -52,6 +52,9 @@
     GIT_DIFF_LINE_CONTEXT: 'normal',
     GIT_DIFF_LINE_ADDITION: 'insert',
     GIT_DIFF_LINE_DELETION: 'delete',
+    GIT_DIFF_LINE_CONTEXT_EOFNL: 'normal-eofnl',
+    GIT_DIFF_LINE_ADD_EOFNL: 'insert-eofnl',
+    GIT_DIFF_LINE_DEL_EOFNL: 'delete-eofnl',
```

Comment:
Where are these new properties referenced?

Reply:
https://github.com/mozilla/addons-server/pull/11912/files#diff-c58b3b043d425ab67d30f4ad957a3b3eR44-R48 defines them and https://github.com/mozilla/addons-server/pull/11912/files#diff-c58b3b043d425ab67d30f4ad957a3b3eR597 actually makes use of them by passing on the `origin` and looking it up in this mapping.